engine/src/main/java/org/terasology/rendering/dag/stateChanges/EnableMaterial.java

```diff
@@ -16,30 +16,32 @@
 package org.terasology.rendering.dag.stateChanges;
 
 import com.google.common.base.Objects;
+import org.terasology.assets.ResourceUrn;
 import org.terasology.registry.CoreRegistry;
 import org.terasology.rendering.ShaderManager;
 import org.terasology.rendering.assets.material.Material;
 import org.terasology.rendering.dag.RenderPipelineTask;
 import org.terasology.rendering.dag.StateChange;
-import org.terasology.utilities.Assets;
+
+import static org.terasology.rendering.dag.AbstractNode.getMaterial;
 
 /**
  * TODO: Add javadocs
  */
 public final class EnableMaterial implements StateChange {
+    private static final ResourceUrn DEFAULT_MATERIAL_URN = new ResourceUrn("engine:prog.default");
 
-    private static final String DEFAULT_MATERIAL_NAME = "DEFAULT";
-    private static EnableMaterial defaultInstance = new EnableMaterial(DEFAULT_MATERIAL_NAME);
+    private static EnableMaterial defaultInstance = new EnableMaterial(DEFAULT_MATERIAL_URN);
 
     private RenderPipelineTask task;
-    private String materialName;
+    private ResourceUrn materialUrn;
```

Comment:
I'm now wondering if we should call this materialId, given that the resource urn uniquely identify a material. What do you think? Could be done in a separate PR though, it's not that critical.